CMakeLists.txt

```diff
@@ -577,5 +577,6 @@ install(FILES
 
 install(FILES
     ${OPAE_SDK_SOURCE}/cmake/modules/OPAEGit.cmake
+    ${OPAE_SDK_SOURCE}/cmake/modules/OPAEPlugin.cmake
```

Comment:
Need to add to .rpm specs and .deb install files.

Reply:
this should be covered by including the cmake installed directory in lib:
```spec
%{_prefix}/lib/opae-%{version}
```